config/locales/en/course/assessment/assessments.yml

```diff
@@ -44,6 +44,7 @@ en:
           requirements: 'Requirements'
           condition_not_satisfied: >
             You must fulfill the following requirements before you can attempt this assessment:
+          time_not_satisfied: 'Wait until %{start_at} for assessment to unlock'
```

Comment:
Wait until `2023-05-11 15:49:03 +0800 ` is not humanized.
Can we humanize this as in how other datetime stamps are displayed in other pages?

Reply:
There is this method format_datetime available in the application_formatters_helper.rb. Can refer there